service-runner.js

```diff
@@ -36,5 +36,5 @@ module.exports = ServiceRunner;
 
 if (module.parent === null) {
     // Run as a script: Start up
-    return new ServiceRunner().run();
+    new ServiceRunner().run();
```

Comment:
new version of JSCS doesn't like it, it's not a function, so it doesn't allow it.